### ISSUE TITLE
TLS failure on OTP/23

### ISSUE THREAD

**Issue by 4DA** [zhyu/nadia#105]
Running Nadia on OTP/23 results in failure with rather strange error message:
```
{:options, {:sslv3, {:versions, [:"tlsv1.2", :"tlsv1.1", :tlsv1, :sslv3]}}}
```

After enabling hackney tracing with :hackney_trace.enable(:max, :io)
I get this:

```
[hackney trace 80 <0.214.0> 2020:05:20 16:00:43 4915] connect error                                                                                                                                                
   Content: [{module,hackney_connect},{line,293}]
```

The issue is likely due to sslv3 being removed in OTP/23
Similar problem: https://elixirforum.com/t/erlang-v-23-0-and-httpoison-error/31615

**Comment by 4tyTwo** [comment]
It seems to be indeed caused by sslv3 being removed in OTP/23, however some libraries still use it. [`hackney`](https://github.com/benoitc/hackney) which is httpoison is based on, has already fixed this, so now we are only blocked by httpoison itself. Once it updates hackney and issues new release, this problem should be fixed.

For now you can either use OTP/22, fork httpoison yourself or override hackney version I guess

**Comment by zhyu** [comment]
published nadia 0.7.0, which includes the fix of hackney.
